assets/src/edit-story/output/element.js

```diff
@@ -61,6 +62,10 @@ function OutputElement({ element }) {
             position: 'absolute',
             top: 0,
             left: 0,
+            borderRadius:
+              borderRadius && canMaskHaveBorder(element)
```

Comment:
Again here you both check if it has a border-radius and if the element supports it. I can't see a situation where it would have the property but not support it.